internal/stage_match_alphanumeric.go

```diff
@@ -1,19 +1,48 @@
 package internal
 
 import (
+	"fmt"
+	"strings"
+
+	"github.com/codecrafters-io/tester-utils/random"
 	"github.com/codecrafters-io/tester-utils/test_case_harness"
 )
 
 func testMatchAlphanumeric(stageHarness *test_case_harness.TestCaseHarness) error {
+
+	words := random.RandomWords(2)
+
+	specialCharacters := []string{"+", "-", "÷", "×", "∑", "∞", "≠", "≥", "≤", "$", "€", "£", "₹", "¥"}
```

Comment:
```suggestion
	specialCharacters := []string{"+", "-", "÷", "×", "$", "€"}
```

(Let's stick to a smaller set of well-known characters)